GliaWidgets/Sources/View/Chat/GVA/GVAPersistentButtonView.swift

```diff
@@ -9,7 +9,7 @@ final class GvaPersistentButtonView: OperatorChatMessageView {
     private let environment: Environment
 
     init(with style: ChatStyle, environment: Environment) {
-        self.viewStyle = style.gva.persistentButton
+        self.viewStyle = style.gliaVirtualAssistant.persistentButton
```

Comment:
There's a SwiftLint issue on this file, at the end

Reply:
fixed [c853494](https://github.com/salemove/ios-sdk-widgets/pull/687/commits/c8534945bc98d88e7f92c7462e2ea6de5c5546bf)